apps/explorer/.env

```diff
@@ -27,3 +27,4 @@ NX_EXPLORER_VALIDATORS=1
 NX_EXPLORER_MARKETS=1
 NX_EXPLORER_ORACLES=1
 NX_EXPLORER_TXS_LIST=1
+NX_EXPLORER_SUPPORTS_SUCCESSOR_MARKETS=0
```

Comment:
suggestion (blocking): Let's use the already existing feature flag for that `NX_SUCCESSOR_MARKETS`